docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+
+1. Create a directory under user home directory 
+
+    `[ec2-user@hostname ~] $ mkdir ~/debezium`
+
+2. Download the sample config scripts from ***https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts*** and copy them into that directory
+
+    `[ec2-user@hostname ~] $ cp *.zip debezium/`
+
+3. Enter into that directory
+
+    `[ec2-user@hostname ~] $ cd debezium`
+
+4. ***export DEBEZIUM_VERSION*** environment variable
+
+    `[ec2-user@hostname ~] $ export DEBEZIUM_VERSION=0.7.1`
+
+5. ***Build docker-compose files*** named docker-compose.yaml (default name)
+
+    `[ec2-user@hostname ~] $ docker-compose build`
+
+6. ***Start the docker*** containers. Applications running on containers are ***Zookeper, Apache Kafka, Debezium Kafka Connector, PostgreSQL db***. 
+
+    `[ec2-user@hostname ~] $ docker-compose up`
+
+7. Create a table named ***customers*** on target or sink PostgreSQL db
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "CREATE TABLE customers (id SERIAL,  first_name VARCHAR(255) NOT NULL,  last_name VARCHAR(255) NOT NULL,  email VARCHAR(255) NOT NULL,  PRIMARY KEY(id));"'`
+
+8. ***Start target PostgreSQL DB connector*** running in a container
+
+    `[ec2-user@hostname ~] $ curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @jdbc-sink.json`
+
+9. ***Start source AWS RDS PostgreSQL DB connector***
+
+    `[ec2-user@hostname ~] $ curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @source-postgres.json`
+
+10. ***Verify*** that the connector definitions are added to the debezium connector
+    
+	`[ec2-user@hostname ~] $ curl -H "Accept:application/json" localhost:8083/connectors/`
+
+11. ***Verify*** that the PostgreSQL database has the same content as in AWS RDS PostgreSQL DB:
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "select * from customers"'`
+
+12. Optionally, if you wanna display the current db name, run the following command 
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "SELECT current_database();"'`
+
+13. Optionally, If you wanna login into PostgreSQL DB container with an of "f86dfa1bba34", run the following command 
+
+    `[ec2-user@hostname ~] $ docker exec -it f86dfa1bba34 bash`
+
+14. Optionally, if you wanna ***restart the servers***, press ***Ctrl +C*** and run the following command again
+
+----
+    [ec2-user@hostname ~] $ export DEBEZIUM_VERSION=0.7.1
+    [ec2-user@hostname ~] $ docker-compose down
+    [ec2-user@hostname ~] $ docker-compose up
+----
+
+### 1.4 Prepare Postgresql Standalone Db Environment As A Source And Target(Sink) At The Same Time For Debezium Connector On Linux
```

Comment:
So I'm a bit lost in that second section. Why is this being used as a source *and* a sink? It'd be nice to explain th use case for this, esp. as you do all those manually setup steps (compiling wal2json etc.) which one normally wouldn't do in RDS. So it'd be good to take a step back here and give some motivation for these steps.

Reply:
As I explained above, the reason is that;
"A database may be a source for a particular topic, and at the same time, the same database may be defined as a sink database for another topic. In our case, we are using micro services, and each service has its own database. However, each database needs some other table data that belongs to an another micro service' database. We used debezium for that purpose. That's, they exchange data among themselves."

I may not mention about "at the same time" stuff. I can split them into 2 pieces. What do you recommend?